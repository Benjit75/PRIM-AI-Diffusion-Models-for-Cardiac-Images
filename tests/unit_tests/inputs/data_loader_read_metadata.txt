training/patient001/Info.cfg
training/patient002/Info.cfg
testing/patient101/Info.cfg
testing/patient102/Info.cfg