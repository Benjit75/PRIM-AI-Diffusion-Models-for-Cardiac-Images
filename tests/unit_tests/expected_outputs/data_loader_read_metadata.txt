{'ED': 1, 'ES': 12, 'Group': 'DCM', 'Height': 184.0, 'NbFrame': 30, 'Weight': 95.0}
{'ED': 1, 'ES': 12, 'Group': 'DCM', 'Height': 160.0, 'NbFrame': 30, 'Weight': 70.0}
{'ED': 1, 'ES': 14, 'Group': 'DCM', 'Height': 169.0, 'NbFrame': 30, 'Weight': 79.0}
{'ED': 1, 'ES': 13, 'Group': 'NOR', 'Height': 156.0, 'NbFrame': 30, 'Weight': 75.0}